src/components/Camera/style.scss

```diff
@@ -112,8 +112,33 @@
   }
 
   .docLiveCaptureFrame & {
-    height: auto;
-    margin: 0;
+    left: 0;
+
+    @media (orientation: portrait) {
+      height: 100%;
+      width: 100%;
+    }
+
+    @media (orientation: landscape) {
+      height: auto;
+      width: 100%;
+      top: 21.5%;
```

Comment:
How did you get to this value?

Reply:
For both instances, through trial & error with different devices. In this case, starting with the default for when the modal is not taking the full width, height of a device (tablets).